includes/admin.php

```diff
@@ -196,11 +210,12 @@ function handle_redirect_saving( $post_id ) {
 	$data = sanitise_and_normalise_redirect_data(
 		wp_unslash( $_POST['hm_redirects_from_url'] ),
 		wp_unslash( $_POST['hm_redirects_to_url'] ),
-		wp_unslash( $_POST['hm_redirects_status_code'] )
+		wp_unslash( $_POST['hm_redirects_status_code'] ),
+		wp_unslash( isset( $_POST['hm_redirects_parameters'] ) ?: '' ),
```

Comment:
Not sure I understand why we want this value to be `true` _or_ an empty string. Should it just be the `isset()` check on its own? You wouldn't need to unslash the value then as it'd just be a boolean